cfgov/v1/management/commands/delete_page.py

```diff
@@ -36,7 +35,7 @@ def handle(self, *args, **options):
             raise CommandError('Must supply a slug or an id')
         try:
             page_to_delete.delete()
-            flush_akamai()
+            page_to_delete.flush()
```

Comment:
Ah, thanks for pointing this out!  Yeah, it looks like it might not work, but it does since the page reference is still intact, and you're just passing along the path to Akamai.  I can't think of a better way to do this, but am totally open to suggestions.